packages/fileutils-windows/fileutils-windows.0.6.1/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ocaml-fileutils"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "git+https://github.com/gildor478/ocaml-fileutils.git"
+bug-reports: "https://github.com/gildor478/ocaml-fileutils/issues"
+doc: "https://gildor478.github.io/ocaml-fileutils/"
+
+build: [
+  ["dune" "build" "-p" "fileutils" "-j" jobs "-x" "windows"]
+  ["dune" "runtest" "-p" "fileutils" "-j" jobs] {with-test}
```

Comment:
As the tests are not testing the windows version but the regular one, we usually remove the with-test targets in this repo (and the with-tests dependencies accordingly). 

(I've never had the case with the doc myself but I would say that the same applies and we leave to the regular package the duty to build the doc if the doc is wanted...)

Reply:
That's also a fair point.